plugins/image-prioritizer/readme.txt

```diff
@@ -27,7 +27,9 @@ The current optimizations include:
    1. Apply lazy loading to `IMG` tags based on whether they appear in any breakpoint’s initial viewport.
    2. Implement lazy loading of CSS background images added via inline `style` attributes.
    3. Lazy-load `VIDEO` tags by setting the appropriate attributes based on whether they appear in the initial viewport. If a `VIDEO` is the LCP element, it gets `preload=auto`; if it is in an initial viewport, the `preload=metadata` default is left; if it is not in an initial viewport, it gets `preload=none`. Lazy-loaded videos also get initial `preload`, `autoplay`, and `poster` attributes restored when the `VIDEO` is going to enter the viewport.
-5. Ensure that [`sizes=auto`](https://make.wordpress.org/core/2024/10/18/auto-sizes-for-lazy-loaded-images-in-wordpress-6-7/) is added to all lazy-loaded `IMG` elements.
+5. Responsive image sizes:
+   1. Ensure [`sizes=auto`](https://make.wordpress.org/core/2024/10/18/auto-sizes-for-lazy-loaded-images-in-wordpress-6-7/) is added to all lazy-loaded `IMG` elements.
```

Comment:
I know this text isn't new, but might be worth clarifying why that's important. Based on how it's currently phrased here, people might argue "But WordPress Core already does this!".

So maybe this can be rephrased to clarify that it's about ensuring that `sizes=auto` is applied correctly after Image Prioritizer has improved/fixed lazy-loading of images as done by WordPress.

Since the `sizes=auto` part isn't really a feature this plugin does better than WordPress (it's the lazy-loading portion it does better than WordPress, but not how it applies `sizes=auto`), I would put the point of computing the correct `sizes` above this, as it's a more powerful feature of the plugin.

Reply:
Like so?

```suggestion
   1. Ensure [`sizes=auto`](https://make.wordpress.org/core/2024/10/18/auto-sizes-for-lazy-loaded-images-in-wordpress-6-7/) is set on `IMG` tags after setting correct lazy-loading (above).
```

Reply:
Yeah, plus moving it to #2 in the list as mentioned :)

Reply:
Ah yes! Done: 4d0251a4396c3e3621a30eacb37854dcba45dd58